cli/pkg/credentialmanager/handler.go

```diff
@@ -175,17 +176,12 @@ func getCurrentContextFromKubeConfig() error {
 	}
 
 	fileContent, err := fileutils.ReadFile(kubeconfig)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Warning: could not open KUBECONFIG file: "+err.Error()+"\n")
-		return nil
-	}
-
-	err = yaml.Unmarshal(fileContent, &kubeConfigFile)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Warning: could not parse KUBECONFIG file: "+err.Error()+"\n")
-		return nil
+		fmt.Println("Hint: If you don't have 'kubectl' installed you can disable this check via 'keptn-local set config KubeContextCheck false'")
```

Comment:
shouldn't this message refer to the kubeconfig file or the kube context instead of `kubectl`?

Reply:
```suggestion
		fmt.Println("Hint: If you don't have 'kubectl' installed you can disable this check via 'keptn set config KubeContextCheck false'")
```